config/auth.php

```diff
@@ -37,7 +37,7 @@
 
     'guards' => [
         'api' => [
-            'driver' => 'token',
+            'driver' => 'session',
```

Comment:
This should be `token`. [Sanctum's guard handles the switch](https://github.com/laravel/sanctum/blob/cf07490b92e07ee6206f712d4502e8a2a41f5049/src/Guard.php#L54-L60) between token and session internally, no need to worry about that.